doc/source/train/user_guide.rst

```diff
@@ -923,59 +874,59 @@ number of retries is configurable through the ``max_retries`` argument of the
 
 .. TODO.
 
-.. _train-profiling:
+.. .. _train-profiling:
```

Comment:
Add a note for this explaining why it's no longer supported (callbacks) and what the long term solution will be (checkpoints)